### ISSUE TITLE
[Bug Report] Error in new version

### ISSUE THREAD

**Issue by ILIKECHICKENTOOMUCH** [Merathilis/ElvUI_MerathilisUI#19]
### What is the issue you are having?

Updated to version 4.95. An error occurs when inviting a party or accepting a party

### What is the expected behavior?

There should be no errors.

### What actually happened?

I get an error message

### Suggested Solution/Workaround

/muierror command 
Only ElvUI (v12.79), ElvUI_MerathilisUI (v4.95)

### Errors

```shell
5x ...\ElvUI_MerathilisUI\Mainline\Modules\Misc\Alerts.lua:100: Usage: RegisterEvent("eventname", "methodname"): 'methodname' - method 'UNIT_SPELLCAST_SUCCEEDED' not found on self.
[string "=[C]"]: ?
[string "@ElvUI\Libraries\Core\Ace\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:112: in function `RegisterEvent'
[string "@ElvUI_MerathilisUI\Mainline\Modules\Misc\Alerts.lua"]:100: in function <...\ElvUI_MerathilisUI\Mainline\Modules\Misc\Alerts.lua:98>
[string "=[C]"]: ?
[string "@ElvUI\Libraries\Core\Ace\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:29: in function <...Core\Ace\CallbackHandler-1.0\CallbackHandler-1.0.lua:25>
[string "@ElvUI\Libraries\Core\Ace\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:64: in function `Fire'
[string "@ElvUI\Libraries\Core\Ace\AceEvent-3.0\AceEvent-3.0-4.lua"]:120: in function <...vUI\Libraries\Core\Ace\AceEvent-3.0\AceEvent-3.0.lua:119>
```

### ElvUI Changelog/Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://github.com/Merathilis/ElvUI_MerathilisUI/blob/development/CHANGELOG.md) and done the [Troubleshooting Steps](https://github.com/Merathilis/ElvUI_MerathilisUI/wiki/Troubleshooting).

**Comment by Repooc** [comment]
Already been addressed in commit 2de2a62  He just hasnt done a release yet with it

**Comment by Merathilis** [comment]
Thanks for reporting it, preparing a release.
